src/americas_essential_data/cli/generate_multipage_status_report.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+from pathlib import Path
+
+import jinja2
+
+from americas_essential_data.resource_monitor.check_history import CheckHistory
+from americas_essential_data.resource_monitor.config import MonitorConfig
+from americas_essential_data.resource_monitor.url_checker import URLChecker
+
+
+def generate_multipage_status_report(config: MonitorConfig, output_path: Path):
+    # Initialize checker and history
```

Comment:
Can we stop redoing the check within the page generation, or is there something that makes it important to keep duplicating the actual check?

Reply:
I fully believe we can separate the URL checks from page generation; however, I haven't dug in to actually make it happen. Maybe something for a future PR 🙂